.github/ISSUE_TEMPLATE/community-question.md

```diff
@@ -13,4 +13,4 @@ A brief description of the question or issue:
 
 You can also join the discord community [here](https://discord.com/invite/jZQs6Wu)
 
-Feel free to check out other cool repositories of Eddie Jaoude Community [here](https://github.com/EddieJaoudeCommunity)
+Feel free to check out other cool repositories of EddieHub Community [here](https://github.com/EddieHubCommunity)
```

Comment:
Looks good 👍🏻 Do you think that we can change ~~EddieHub Community~~ to `EddieHub`

Reply:
I think `EddieHub` alone is better. But this is still an improvement on what there is currently. A new PR can come in to make it all consistent, but I am keen to get this in, so that the links work again